torch/_library/custom_ops.py

```diff
@@ -332,12 +332,16 @@ def backend_impl(*args, **kwargs):
                                 fn = self._backend_fns[device_type]
                                 module = inspect.getmodule(fn)
                                 raise RuntimeError(
-                                    f"Tensors returned from custom ops (1) must not "
-                                    f"be inputs to the custom op and (2) may not alias "
-                                    f"any inputs or other returns. Please clone the "
-                                    f"the offending output tensors (e.g. output.clone()) "
-                                    f"or refactor your code. "
-                                    f"Offending op: {self._name} (with implementation in {module})"
+                                    f"{self._name} (with implementation in {module}): "
```

Comment:
Aren't inplace ops expected to return the input that they modify inplace as-is though to be able to support autograd?

Reply:
We don't support that in the compile path or the torch.library.custom_op API